packages/web/src/app/admin/products/SortableHeader.component.tsx

```diff
@@ -0,0 +1,43 @@
+"use client";
+
+import React from "react";
+
+// Define the type for sort direction.
+type SortDirection = "asc" | "desc";
+
+type SortIndicatorProps = {
+    isSorted: boolean;
+    direction: "ascending" | "descending";
+};
```

Comment:
Use a `boolean` or `enum` value here.

Also `SortIndicatorProps.direction` could be of type `SortDirection`. (Maybe declare SortDirection in a global way to also use this in other components? I saw that already in multiple files)